package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "bare-readline",
-  "version": "1.0.9",
+  "version": "1.0.10",
```

```suggestion
  "version": "1.0.9",
```

The version must only be bumped on `main` just prior to release.

---

oh gotcha ill change the version back too sorry!